ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-master.yaml

```diff
@@ -3,10 +3,34 @@ base_images:
     name: "4.9"
     namespace: ocp
     tag: base
+  driver-operand:
+    name: "4.9"
+    namespace: ocp
+    tag: aws-efs-csi-driver
+  livenessprobe-operand:
+    name: "4.9"
+    namespace: ocp
+    tag: csi-livenessprobe
   ocp_builder_rhel-8-golang-1.16-openshift-4.9:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.16-openshift-4.9
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.8
```

Comment:
Just to be sure: is `4.8` expected here? 
A comment would be nice, if it survives `make update`.

Reply:
it does survive `make update`. The reason I was using is v4.8, because v4.9 does not exist and also if you see every use of this image, they all seem to be using `v4.8` cc @emilvberglind @stevekuznetsov